src/sqlfluff/rules/L033.py

```diff
@@ -11,8 +11,8 @@ class Rule_L033(BaseRule):
     """``UNION [DISTINCT|ALL]`` is preferred over just ``UNION``.
 
     .. note::
-       This rule is only enabled for dialects that support ``UNION DISTINCT``
-       (``ansi``, ``bigquery``, ``hive``, ``mysql``, and ``redshift``).
+       This rule is only enabled for dialects that support ``UNION`` and
+       ``UNION DISTINCT`` (``ansi``, ``hive``, ``mysql``, and ``redshift``).
```

Comment:
Using `UNION` without `DISTINCT` or `ALL` is invalid in BigQuery and so will now cause a parsing error. Therefore this rule no longer is relevant for BigQuery.